src/main/java/org/cryptomator/integrations/keychain/KeychainAccessProvider.java

```diff
@@ -45,6 +57,18 @@
 	 */
 	void changePassphrase(String key, CharSequence passphrase) throws KeychainAccessException;
```

Comment:
Add a `@Deprecated` annotation to the method and a `@deprecated Please use {@link #changePassphrase(String, String, CharSequence} instead` to the JavaDoc, so we can eventually reverse the overloading and make the three-param variant the default.